bin/desi_fit_guide_star_coordinates

```diff
@@ -24,12 +24,15 @@ log   = get_logger()
 
 fm = FieldModel()
 
-header  = pyfits.open(args.fits_header)[1].header
+header  = pyfits.open(args.fits_header)[0].header
 fm.ra  = header["TARGTRA"]
```

Comment:
This change is surprising to me.  Does it work over the full range of exposures we have, or are we in a situation where some gfa_proc results have the headers we need in HDU 0 while others have it in HDU 1?

Reply:
guide files, like guide-00047450.fits.fz have the info in HDU=0. other files, like gfa-00042972.fits.fz have it in HDU=1. I will change the code so that both work.